src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inverseObj = {};
```

Comment:
Don't use names of data types in variable name
```suggestion
  const inversedRobot = {};
```